TomEE-8.0/jdk17/microprofile/Dockerfile

```diff
@@ -41,20 +41,23 @@ RUN set -xe; \
   626C542EDA7C113814B77AF09C04914D63645D20 \
   # Jean-Louis Monteiro (CODE SIGNING KEY) <jlmonteiro@apache.org>
   3948829384B269D333CC5B98358807C52B4B0E23 \
+  # Richard Zowalla <richard.zowalla@hs-heilbronn.de>
```

Comment:
My code signing key is correct.

I would use the `uid` of the key for the comment, which is:

```
Richard Zowalla (Code Signing Key) <rzo1@apache.org>
```